File: client_manage_interface/src/main/java/cn/edu/aynu/rjxy/management/service/ShowingsService.java
package cn.edu.aynu.rjxy.management.service;


import cn.edu.aynu.rjxy.pojo.TbShowings;
import com.github.pagehelper.PageInfo;

import java.util.Date;
import java.util.List;

/**
 * 服务层接口
 * @author Administrator
 *
 */
public interface ShowingsService {

	/**
	 * 返回全部列表
	 * @return
	 */
	public List<TbShowings> findAll();
	
	
	/**
	 * 返回分页列表
	 * @return
	 */
	public PageInfo findPage(int pageNum, int pageSize);


	/**
	 * 增加
	*/
	public void add(String username,TbShowings showings);


	/**
	 * 修改
	 */
	public void update(TbShowings showings);


	/**
	 * 根据ID获取实体
	 * @param id
	 * @return
	 */
	public TbShowings findOne(Long showId);


	/**
	 * 批量删除
	 * @param ids
	 */
	public void delete(Long[] showIds);

	/**
	 * 分页
	 * @param pageNum 当前页 码
	 * @param pageSize 每页记录数
	 * @return
	 */
	public PageInfo findPage(TbShowings showings, int pageNum, int pageSize);


}

File: client_seller_web/src/main/webapp/js/service/showingsService.js
//服务层
app.service('showingsService',function($http){
	    	
	//读取列表数据绑定到表单中
	this.findAll=function(){
		return $http.get('../showings/findAll.do');		
	}
	//分页 
	this.findPage=function(page,rows){
		return $http.get('../showings/findPage.do?pageNum='+page+'&pageSize='+rows);
	}
	//查询实体
	this.findOne=function(id){
		return $http.get('../showings/findOne.do?id='+id);
	}
	//增加 
	this.add=function(entity){
		return  $http.post('../showings/add.do',entity );
	}
	//修改 
	this.update=function(entity){
		return  $http.post('../showings/update.do',entity );
	}
	//删除
	this.dele=function(ids){
		return $http.get('../showings/delete.do?ids='+ids);
	}
	//搜索
	this.search=function(page,rows,searchEntity){
		return $http.post('../showings/search.do?pageNum='+page+"&pageSize="+rows, searchEntity);
	}    	
});